web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file 
+ * Generate markup to be displayed. Functionality in this controller is
+ * wired to Drupal in mymodule.routing.yml.
+ */
+
+namespace Drupal\mymodule\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+
+class FirstController extends ControllerBase {
+
+  public function simpleContent() {
+    return [
+      '#type' => 'markup',
+      '#markup' =>('hello world'),
```

Comment:
Why hello world in brackets?

Reply:
I was using the 't' function, at that time put the brackets , however  when I removed the t function I forgot to remove it.
Removed Now.